src/templates/fr/2019/contributors.html

```diff
@@ -302,14 +302,23 @@ <h1 class="title title-lg">
     <div class="contributors-count"><span id="filtered-contributors">{{ config.contributors.items() | length }}</span> personnes</div>
     <div class="filter-container">
       <p>Filtrer par équipe&nbsp;:</p>
-      <button class="analysts" type="button">Analyse</button>
-      <button class="authors" type="button">Rédaction</button>
-      <button class="brainstormers" type="button">Réflexion</button>
-      <button class="designers" type="button">Design</button>
-      <button class="developers" type="button">Développement</button>
-      <button class="editors" type="button">Édition</button>
-      <button class="reviewers" type="button">Relecture</button>
-      <button class="translators" type="button">Traduction</button>
+      {% set localizedTeamNames = {
+            "analysts": "Analyse",
+            "authors": "Rédaction",
+            "brainstormers": "Réflexion",
+            "designers": "Design",
+            "developers": "Développement",
+            "editors": "Édition",
+            "reviewers": "Relecture",
```

Comment:
Examinateur ?

Reply:
L'idée ici est de basculer autant que possible sur les nom des actions plutôt que sur les noms des postes, afin de rester inclusif. De plus, examen ne correspond pas à cette tâche, qui correspond dans un processus éditorial à de la relecture par des pairs.

Reply:
> L'idée ici est de basculer autant que possible sur les nom des actions plutôt que sur les noms des postes, afin de rester inclusif. De plus, examen ne correspond pas à cette tâche, qui correspond dans un processus éditorial à de la relecture par des pairs.

Si tu bascules à l'action de 're-lire' c'est comme une action de voir sans interventions aprés

Reply:
Non ce n'est pas la définition de ce verbe. Relire implique une intention critique, une volonté de correction. C'est un terme courant du monde de l'édition.

The same remark can be made for "to review", which literally means "to view again". However, in English, the corrective purpose is clear. It's the same in French.

Reply:
Some FR définitions: [fr-FR](https://www.larousse.fr/dictionnaires/francais/relecture/67873), [fr-CA](https://www.cnrtl.fr/definition/relecture), [wiktionary](https://fr.wiktionary.org/wiki/relecture)…